lib/msal-browser/src/broker/nativeBroker/NativeMessageHandler.ts

```diff
@@ -91,11 +103,15 @@ export class NativeMessageHandler {
             channel: NativeConstants.CHANNEL_ID,
             extensionId: this.extensionId,
             responseId: this.responseId++,
-
             body: {
                 method: NativeExtensionMethod.HandshakeRequest
             }
         };
+        this.handshakeEvent.addStaticFields({
+            wamChannel: NativeConstants.CHANNEL_ID,
+            wamExtensionId: this.extensionId,
```

Comment:
Just extensionId or nativeExtensionId
```suggestion
            extensionId: this.extensionId,
```